src/timer/mod.rs

```diff
@@ -83,7 +86,7 @@ pub fn execute_and_measure(mut command: Command) -> Result<TimerResult> {
         use std::os::windows::process::CommandExt;
 
         // Create a suspended process
-        command.creation_flags(4);
+        command.creation_flags(CREATE_SUSPENDED);
```

Comment:
:+1: 

Maybe we can remove the comment now? (Or explain what a suspended process is?)

Reply:
I hope the explanation helps.